src/main/java/com/gitblit/transport/ssh/SshDaemon.java

```diff
@@ -133,7 +133,7 @@ public SshDaemon(IGitblit gitblit, WorkQueue workQueue) {
 		//Will do GSS ?
 		GSSAuthenticator gssAuthenticator = null;
 		if(settings.getBoolean(Keys.git.sshWithKrb5, false)) {
-			gssAuthenticator = new SshKrbAuthenticator(gitblit);
+			gssAuthenticator = new SshKrbAuthenticator(gitblit, settings.getBoolean(Keys.git.sshKrb5StripDomain, false));
```

Comment:
Instead of passing a flag let's pass the IStoredSettings object and then retrieve the setting while inside the constructor.  This will be more flexible for the future.